tests/output/ConsoleInteractionTest.py

```diff
@@ -174,6 +175,44 @@ def test_require_settings(self):
 
             self.assertEqual(generator.last_input, 2)
 
+    def test_print_bears(self):
+        bears = inverse_dicts(self.local_bears, self.global_bears)
+        expected = 'No bears to show. Did you forget to install '\
+                   'the `coala-bears` package?'\
+                   ' Try `pip3 install coala-bears`.\n'
+        with retrieve_stdout() as stdout:
+            print_bears({}, True, True, self.console_printer)
```

Comment:
Also pass a dict to the first argument, see https://github.com/coala/coala/blob/master/tests/output/ConsoleInteractionTest.py#L729-L730